src/diffusers/models/lora.py

```diff
@@ -89,6 +96,52 @@ def __init__(self, *args, lora_layer: Optional[LoRAConv2dLayer] = None, **kwargs
     def set_lora_layer(self, lora_layer: Optional[LoRAConv2dLayer]):
         self.lora_layer = lora_layer
 
+    def _fuse_lora(self):
+        if self.lora_layer is None:
+            warnings.warn("Calling fuse_lora() is not supported. It will be a no-op.", RuntimeWarning)
```

Comment:
I think it's sensible to do it here because when we train LoRA in `diffusers` and the LoRA is in the `diffusers` format from the get-go (like this [one](https://huggingface.co/sayakpaul/lora-trained-xl)).

Reply:
We treat the LoRA delegation differently from how it's done when we try to load Kohya-style or any other non-diffusers LoRAs.

Reply:
Sounds good to me!